src/codebase.md

```diff
@@ -260,3 +260,41 @@ crates optimally for your semantic needs.
 | [ring](https://docs.rs/ring/)           | the leading crypto library         |
 | [nalgebra](https://docs.rs/nalgebra/)   | linear algebra                     |
 | [once_cell](https://docs.rs/once_cell/) | complex static data                |
+
+## How should I call C++ functions from Rust and vice-versa?
+
+Behind the scenes, Rust natively supports calling C functions using `extern "C"`,
```

Comment:
"Behind the scenes" -- not sure this image works here. These are directly accessible features -- they're just low-level and unsafe.

Maybe "At the lowest level" instead of "Behind the scenes"?

Reply:
Yes, I agree, I'm not sure how those words ended up there :) I just deleted them